controllers/componentdetectionquery_controller.go

```diff
@@ -64,7 +66,8 @@ type ComponentDetectionQueryReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.2/pkg/reconcile
 func (r *ComponentDetectionQueryReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log := r.Log.WithValues("ComponentDetectionQuery", req.NamespacedName)
+	log := r.Log.WithValues("ComponentDetectionQuery", req.NamespacedName).WithValues("clusterName", req.ClusterName)
```

Comment:
wonder if you only want to add the clustername in the controller files? Other files that use logs will not have this value..

Reply:
Good catch. We should have it for all of the cases where we create a logger. Will need to pass in the request struct into those functions

Reply:
@maysunfaisal Updated.